dozer/cogs/roles.py

```diff
@@ -221,7 +221,16 @@ async def giveme(self, ctx, *, roles):
             e.add_field(name='{} role(s) could not be found!'.format(extra),
                         value='Use `{0.prefix}{0.invoked_with} list` to find valid giveable roles!'.format(ctx),
                         inline=False)
-        await ctx.send(embed=e)
+        msg = await ctx.send(embed=e)
+        await msg.add_reaction("❌")
```

Comment:
This needs error handling in case Dozer doesn't have Add Reactions permissions, or the user has blocked the bot.